test/require-config.js

```diff
@@ -6,30 +6,31 @@ require.config({
     baseUrl: '/base',
     waitSeconds: 60,
     paths: {
-        backbone: 'node_modules/backbone/backbone',
+        'backbone': 'node_modules/backbone/backbone',
```

Comment:
We're currently set to `consistent`, but I got confused and actually documented the `consistent-as-needed` behavior I think? So maybe that's what we should go with because that's what people already thumbsed up on that PR.

(Here, I think the linter is tripping on `backbone.paginator`, which isn't a valid property name without quotes, so they're all being quoted. But if `backbone.paginator` wasn't here, and we were using `consistent-as-needed`, none of these would need quotes.)